CHANGES.md

```diff
@@ -1,5 +1,30 @@
 # Changes
 
+## unreleased
+
+- Remove unmaintained `chrono` dependency
+
+### Breaking Changes
+
+Due to security issues caused by the unmaintained `chrono` crate
```

Comment:
`chrono` is maintained again. It's fine to remove it (or switch to `time`), but we shouldn't be saying it's unmaintained or insecure.

Reply:
ok, done.